src/commit.ts

```diff
@@ -3,4 +3,5 @@ export interface Commit {
   author: string;
   subject: string;
   body?: string;
+  tag: string;
```

Comment:
Not sure how I feel about forcing every commit to have a tag... there has to be a way around it. Definitely a later sort of item but documenting it now in case I forget.